flow.md

```diff
@@ -0,0 +1,84 @@
+# Work-flow
+
+## Repositories
+| Type | Description |
+| ---| ---| ---| 
+| NETN FOM  | Main Repository for the entire FOM including all related FOM Modules.|
+| NETN-* | Repository for a specific FOM Module.|
+
+## Branches
+
+| Type | Description |
+| ---| ---|
+| Master Branch |The latest officially released version. |
+| Develop Branch |The lastest development version with reviewed changes. |
+| Release Branches |Created to prepare for new official release. |
+| Maintenance Branches |Opened to maintain specific baseline or older version version. |
+| Edit Branches |Openend to develop proposed updates and closed after pull-request review and approval.|
```

Comment:
Suggest: "Temporary, used to develop..."

Reply:
Fixed as suggested